src/test/java/programminglife/gui/GuiControllerTest.java

```diff
@@ -35,7 +35,6 @@
         stage.show();
         stage.toFront();
         File f = new File(getClass().getResource("/test.gfa").getPath());
-        Graph g = Graph.parse(f);
```

Comment:
Redundant for now, as it is handled in the pull request from the GuiConrollerTest